psydac/feec/global_projectors.py

```diff
@@ -659,7 +659,7 @@ class Projector_H1vec(GlobalProjector):
     
     Parameters
     ----------
-    H1vec : ProductFemSpace
+    H1vec : FemSpace
```

Comment:
I think `H1vec` should be an object of type `VectorFemSpace`. Or not?

Reply:
(I changed the description)